test/lib/testUtils.ts

```diff
@@ -17,8 +17,21 @@ import {
 } from "@nmshd/connector-sdk";
 import fs from "fs";
 import { DateTime } from "luxon";
+import { MongoClient } from "mongodb";
 import { ValidationSchema } from "./validation";
 
+export async function connectAndEmptyCollection(databaseName: string, collectionName: string): Promise<void> {
+    const client = new MongoClient(process.env.DATABASE_CONNECTION_STRING!);
+    try {
+        await client.connect();
+        const database = client.db(databaseName);
+        const collection = database.collection(collectionName);
+        await collection.deleteMany({});
+    } finally {
+        await client.close();
+    }
```

Comment:
you could do that using our database lib, so that you don't have to add mongodb as a dependency